warehouse/templates/manage/token.html

```diff
@@ -62,10 +61,32 @@ <h2>Token for "{{ macaroon.description }}"</h2>
 
       {{ confirm_modal(title=title, confirm_name="Username", confirm_string=user.username, confirm_button_label=confirm_button_label, slug=slug, extra_fields=extra_fields, custom_warning_text=token_warning_text, confirm_string_in_title=False) }}
     </div>
-    <hr>
+
+    <h2>Using this token</h2>
+
+    <p>To use this API token:</p>
+
+    <ul>
+      <li>Set your username to <code>__token__</code></li>
+      <li>Set your password to the token value, including the <code>pypi-</code> prefix</li>
+    </ul>
+
+
+    {% if macaroon.caveats.permissions == "user" %}
```

Comment:
We _might_ want to display this config sample unconditionally, not just for user-scoped tokens. Not 100% sure, though, since project-scoped tokens will primarily apply in CI and other automated setups.

Reply:
I was thinking of adding an else statement with alternative instructions for project specific tokens.
I'm waiting on a response to https://github.com/pypa/warehouse/issues/6551#issuecomment-526480764 to see what this should say :)

Reply:
Gotcha! I'll respond with some notes there 😄